content/docs/get-started/azure/deploy-stack.md

```diff
@@ -49,7 +49,7 @@ Updating (dev):
  +   └─ azure:storage:Account     storage         created
 
 Outputs:
-    connectionString: "DefaultEndpointsProtocol=https;AccountName=storagec10b9cad;AccountKey=f5JxKN8M7mECDlzdB9zTwfJWSplo8jFTFFKRTzGAldscILf1ftrJPaspSA69tzLe24WBbWJ9yTu+mzjaqmPEew==;EndpointSuffix=core.windows.net"
+    connectionString: "DefaultEndpointsProtocol=https;AccountName=storage1b3018e9;AccountKey=...;EndpointSuffix=core.windows.net"
```

Comment:
Good call reconciling the connection strings between the steps: they must be equal indeed